hwdb/70-mouse.hwdb

```diff
@@ -342,8 +342,6 @@ mouse:usb:v046dp402d:name:Logitech M560:
 mouse:usb:v046dpc52b:name:Logitech Unifying Device. Wireless PID:402d:
  MOUSE_DPI=1000@125
 
-# Logitech V220 Cordless Optical Mouse
-mouse:usb:v046dpc51b:name:Logitech USB Receiver:
```

Comment:
According to http://support.logitech.com/en_us/product/v220-cordless-optical-mouse-for-notebooks it seems the mouse is using a pre-version of the small unifying receiver we know now. If there are 2 mice with the same receiver, that means that the values should both be dropped IMO.

I can't find in the docs if c51b would be compatible with hid-logitech-dj.ko, but I am not sure either if it is worth working on those old HW.